Fetch dataset from Spine Generic

PaulBautin opened the pull request:
"csa-atrophy" test dataset is no longer hosted on "openneuro" but on "spine generic project", https://github.com/spine-generic/data-multi-subject. An update is required to reflect this change.

Done: 
- updated README
- remove csa_fetch_dataset
- use data-multi-subject last release https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801

FIX #26

PaulBautin commented:
Ok, are we settling on https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801 for csa-atrophy? @jcohenadad

jcohenadad commented on README.md:
```diff
@@ -30,18 +30,15 @@ Download (or git clone) this repository:
-  - To fetch specific version of openneuro repository (version: 1.0.5, Files: 2501, Size: 7.9GB, Subjects: 248) follow instructions to set openeuro CLI: https://www.npmjs.com/package/openneuro-cli
-  ~~~
-  openneuro download --snapshot 1.0.5 ds001919 data
+  curl -o spinegeneric.zip -L https://github.com/spine-generic/data-multi-subject/archive/master.zip
```
should point to specific release

jcohenadad commented:
> Ok, are we settling on https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801 for csa-atrophy? @jcohenadad

for now yes, when the release is updated, it's easy to update the URL here

PaulBautin commented:
I think PR can be closed @jcohenadad . Most important features:
 - update README so that users download dataset from spine generic  https://github.com/spine-generic/data-multi-subject/releases/tag/r20200801
- remove fetch_dataset script

jcohenadad commented on README.md:
```diff
@@ -19,29 +19,27 @@ Dataset/
 Run the following script within the Dataset folder to extract CSA. This script can be run on desired subjects using flag -include and in parallel processing using flag -jobs.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results process_data.sh
+sct_run_batch -path-data data-multi-subject-master -path-output csa_atrophy_results process_data.sh
```
are you sure this is the correct folder name now that you've changed the url to "https://github.com/spine-generic/data-multi-subject/archive/r20200801.zip"?

PaulBautin replied:
`sct_run_batch` now uses a config file that is not present in this PR (i have implemented this comment as a reminder in commit https://github.com/sct-pipeline/csa-atrophy/pull/28/commits/57b8c1d7ebdb74573131a4634fead5aa3f4243a4).

jcohenadad commented on README.md:
```diff
@@ -57,5 +55,5 @@ After running the analysis, check your Quality Control (QC) report by opening th
 It is now possible to re-run the whole process, pointing to the manual corrections. With the command below labeling will use the manual corrections present in 'seg_manual', otherwise labeling will be done automatically.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
+sct_run_batch -path-data data-multi-subject-master -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
```
same as https://github.com/sct-pipeline/csa-atrophy/pull/28/files#r464656202

PaulBautin replied:
same as > `sct_run_batch` now uses a config file that is not present in this PR (i have implemented this comment as a reminder in commit [57b8c1d](https://github.com/sct-pipeline/csa-atrophy/commit/57b8c1d7ebdb74573131a4634fead5aa3f4243a4)).

jcohenadad commented:
@PaulBautin the PR body seems unfinished and contains an unresolved "TODO". Please cleanup before we merge

jcohenadad commented on README.md:
```diff
@@ -19,29 +19,29 @@ Dataset/
 Run the following script within the Dataset folder to extract CSA. This script can be run on desired subjects using flag -include and in parallel processing using flag -jobs.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results process_data.sh
+sct_run_batch -path-data spinegeneric_r20200801 -path-output csa_atrophy_results process_data.sh
```
sct_run_batch now uses -task flag

jcohenadad replied:
same thing below

PaulBautin replied:
I just did a pull from SCT repo, i don't see flag -task in sct_run_batch. However, i'm thinking of using flag -config so no need for other flags.

jcohenadad replied:
it was actually changed to -script: 
https://github.com/neuropoly/spinalcordtoolbox/blob/master/scripts/sct_run_batch.py#L143

it is always important to test the code that you write

PaulBautin commented:
I have corrected path to dataset in the README and added a comment on users options to test the spine-generic dataset without processing all subjects.

jcohenadad commented on README.md:
```diff
@@ -57,5 +57,5 @@ After running the analysis, check your Quality Control (QC) report by opening th
 It is now possible to re-run the whole process, pointing to the manual corrections. With the command below labeling will use the manual corrections present in 'seg_manual', otherwise labeling will be done automatically.
 ~~~
-sct_run_batch -path-data data -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
+sct_run_batch -path-data spinegeneric_r20200801 -path-output csa_atrophy_results_corrected -path-segmanual seg_manual process_data.sh
```
same here-- that will crash-- please test your procedure, then let me know when your code is ready for a review

PaulBautin commented:
@jcohenadad, i will close this PR without merging it, if it is ok with you? The scope is small and the branch is behind master, i am not able to test procedure easily. I have already merged this PR in #31.
